src/lib/fields.ts

```diff
@@ -0,0 +1,320 @@
+import { IDENTIFIER } from './extension';
+
+/**
+ * Link Pull Request to Aha Record
+ *
+ * @param PR
+ * @param record
+ */
+export const linkPullRequestToRecord = async (PR: AzureDevops.PR, record: Aha.RecordUnion) => {
```

Comment:
@sorza-jimmy I'm seeing exceptions in my test instance from this method. It seems that something is calling this with a null `record`.